JenkinsfileStressTests

```diff
@@ -0,0 +1,83 @@
+def maxRetries = 120
+def waitForCommandSuccess(command, maxRetries) {
+    def tries = 0;
+    while (tries++ < maxRetries) {
+        try {
+            sh command
+            return true
+        } catch (Exception e) {
+            if (tries >= maxRetries) {
+                throw new Exception("${maxRetries} tries exceeded. ${command} failed.", e)
+            }
+            sh "sleep 1"
+        }
+    }
+}
+
+pipeline {
+    agent { label 'my-slave' }
+
+    tools {
+        maven 'maven'
+    }
+
+    stages {
+        stage('Compile') {
+            steps {
+                sh "mvn clean install -DskipTests"
+            }
+        }
+        stage('Build image') {
+            steps {
+                script {
+                    docker.withTool('docker') {
+                        sh "cp server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.jar dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "cp -r server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.lib dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "cp server/container/cli/target/james-server-cli.jar dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "cp -r server/container/cli/target/james-server-cli.lib dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "docker build -t james_run dockerfiles/run/guice/cassandra-rabbitmq"
+                    }
+                }
+            }
+        }
+        stage('Start James') {
+            steps {
+                script {
+                    docker.withTool('docker') {
+                        sh "docker run -d --name=cassandra cassandra:3.11.3"
+                        sh "docker run -d --name=rabbitmq rabbitmq:3.7.7-management"
+                        sh "docker run -d --name=elasticsearch elasticsearch:2.4.6"
+                        try {
+                            waitForCommandSuccess("docker exec cassandra cqlsh -e 'describe columnfamily system.local;'", maxRetries)
```

Comment:
we should not need to wait, at all, james should handle that situation gracefully

Reply:
Yes, I've listed that in the future tasks.
I think it may need some work, depending on the product you want to use.

Reply:
I'm pretty sure it works right now, could you just try?